cmds/ps/ps.go

```diff
@@ -168,17 +177,31 @@ func (pT *ProcessTable) PrepareString() {
 	pT.fstring = fstring
 }
 
+func isPermutation(check string, ref string) bool {
+	checkArray := strings.Split(check, "")
+	refArray := strings.Split(ref, "")
+
+	sort.Strings(checkArray)
+	sort.Strings(refArray)
+
+	return reflect.DeepEqual(check, ref)
+}
+
 // For now, just read /proc/pid/stat and dump its brains.
 // TODO: a nice clean way to turn /proc/pid/stat into a struct. (trying now)
```

Comment:
This TODO was already done when I wrote this version of `ps`, but I forgot to remove it. This can be deleted?

ps.: although I don't have sure if the way I did it is "clean" enough.